packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseDetailsEditor.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('ReleaseDetailsEditor', () => {
       () => {
         expect(useReleaseOperations().updateRelease).toHaveBeenCalledWith(release)
       },
-      {timeout: 250},
+      {timeout: 10_000},
```

Comment:
had to increase timeout here, likely due to coverage instrumentation making things run slower.